.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Do you know why this file is being added?

Reply:
Not sure, but I figured it doesn't hurt. It pertains to the linter, which we want run on all the scripts.